apps/demo/src/app/core/sidenav/sidenav.component.html

```diff
@@ -27,22 +27,27 @@ <h1>Manifest</h1>
         href="https://www.nb.no/items/f080d07ddce0f9ebd2615c44e040197d?manifest=https://api.nb.no/catalog/v1/iiif/f080d07ddce0f9ebd2615c44e040197d/manifest"
         target="_blank"
       >
-        <img
-          src="assets/logo-iiif.png"
-          alt="IIIF Drag-n-drop"
-          class="logo-iiif"
-        /><span>Drag and drop</span>
+        <div class="flex flex-row items-center">
+          <img
+            src="assets/logo-iiif.png"
+            alt="IIIF Drag-n-drop"
+            class="logo-iiif"
+          /><div>DnD</div>
+        </div>
       </a>
       <a
         mat-list-item
+        class="flex flex-row"
```

Comment:
I don't think this is needed? Cannot see any effect by using it, and none of the other anchor elements has it.

Reply:
Fixed